Sources/SwiftGRPCNIO/CallHandlers/BidirectionalStreamingCallHandler.swift

```diff
@@ -36,9 +36,13 @@ public class BidirectionalStreamingCallHandler<RequestMessage: Message, Response
     }
   }
 
-  public override func endOfStreamReceived() {
+  public override func endOfStreamReceived() throws {
     eventObserver?.whenSuccess { observer in
       observer(.end)
     }
   }
+  
+  override func sendErrorStatus(_ status: GRPCStatus) {
+    context?.statusPromise.fail(error: status)
```

Comment:
Where are we `write`-ing the status now?

Reply:
The call contexts schedule closures writing the status off the corresponding promises; see https://github.com/grpc/grpc-swift/blob/master/Sources/SwiftGRPCNIO/ServerCallContexts/UnaryResponseCallContext.swift and https://github.com/grpc/grpc-swift/blob/master/Sources/SwiftGRPCNIO/ServerCallContexts/StreamingResponseCallContext.swift.

Reply:
Ah of course, thanks!